helix-core/src/indent.rs

```diff
@@ -433,17 +433,17 @@ fn query_indents(
 ///     after pos were moved to a new line.
 ///
 /// The indentation is determined by traversing all the tree-sitter nodes containing the position.
-/// Each of these nodes produces some [AddedIndent] for:
+/// Each of these nodes produces some [Indentation] for:
 ///
 /// - The line of the (beginning of the) node. This is defined by the scope `all` if this is the first node on its line.
 /// - The line after the node. This is defined by:
 ///   - The scope `tail`.
 ///   - The scope `all` if this node is not the first node on its line.
 /// Intuitively, `all` applies to everything contained in this node while `tail` applies to everything except for the first line of the node.
 /// The indents from different nodes for the same line are then combined.
-/// The [IndentResult] is simply the sum of the [AddedIndent] for all lines.
+/// The [Indentation] is simply the sum of the [Indentation] for all lines.
```

Comment:
I'm not very familiar with this part of the code but I think this line doesn't apply anymore and could be removed. It looks like the IndentResult and AddedIndent items may have been merged together.

Reply:
There is still a sum of indentations for all lines, (see lines 553, 554).
Added the word result to make it more clear what the sum is.